control_plane/src/pageserver.rs

```diff
@@ -247,7 +250,10 @@ impl PageServerNode {
     }
 
     fn pageserver_env_variables(&self) -> anyhow::Result<Vec<(String, String)>> {
-        Ok(if self.env.pageserver.auth_type != AuthType::Trust {
+        // FIXME: why is this tied to mgmt auth type? Whether or not the safekeeper
```

Comment:
It is the pageserver using a token to access safekeeper. This is used for walreceiver connection.

Additionally there is no AuthType stored directly in env, so pageserver's value most probably is used to turn on the auth globally